ci-bootstrap/README.md

```diff
@@ -0,0 +1,15 @@
+# Resource for setting up role in a new AWS account.
+
+When setting up a new infrastructure for the lambda, first run deploy.sh script to create a correct role with correctly assigned policies.
+
+Replace ENVIRONMENT variable ``` <accountname> ``` with sandbox, cicd, staging or prod
+
+
+Run in the terminal
+cd ci-bootstrap
```

Comment:
Example code outside of the markdown code block

See the readme in https://github.com/ONSdigital/statement-of-record-terraform/pull/22/files for an example set of instructions